src/validation/rules/lib_package.rs

```diff
@@ -8,10 +8,6 @@ impl Rule for LibPackage {
         "Make sure that lib packages don't contain service definitions."
     }
 
-    fn rule_level(&self) -> Level {
```

Comment:
What's the implication / reason behind this change?

Reply:
That the default impl of `Rule` already returns `Level::Error`, so going with the default and not writing code that has no effect beyond the default.